src/main/java/com/deloitte/elrr/aggregator/rules/ProcessCompetency.java

```diff
@@ -136,8 +136,6 @@ private Competency processCompetency(final Activity activity) {
             }
 
         } catch (AggregatorException e) {
```

Comment:
these are another case like last week where if all you do is throw, there is no reason for the try/catch. Same effect as not catching.